Blockzilla/Modules/WebView/WebViewController.swift

```diff
@@ -39,7 +39,7 @@ protocol WebControllerDelegate: class {
 class WebViewController: UIViewController, WebController {
     weak var delegate: WebControllerDelegate?
 
-    private var browserView = WKWebView()
+    var browserView = WKWebView()
```

Comment:
This stands out as no bueno - can we maintain the privacy modifier on this important object?

Reply:
I have a couple of ideas on how we can solve for this. I'll try to connect with @bkmunar after lunch